schema/verify/migrations/insert_json_schema_form_data.sql

```diff
@@ -0,0 +1,13 @@
+-- Verify cif:migrations/insert_json_schema_form_data on pg
+
+begin;
+
+do $$
+  begin
+    assert (
+      (select count(*) from cif.form) > 0
```

Comment:
Should we count the exact number of rows we want to insert?

Reply:
We could, but this will cause `sqitch verify` to fail in the future if we add/remove rows

Reply:
That would be to make sure we insert the proper number of rows and force us to update the verify script if we rework the migration